modules/ROOT/pages/bare-metal.adoc

```diff
@@ -19,14 +19,16 @@ To install FCOS onto bare metal with an ISO, follow these steps:
 NOTE: You can install in either legacy boot (BIOS) mode or in UEFI mode, regardless of what mode the OS will use once installed.
 +
 . Burn the ISO to disk and boot it.
-. From the boot menu, press `<TAB>` (isolinux) or `e` (grub) to edit the kernel command line.
-. Add the following parameters to the kernel command line:
-* `coreos.inst.install_dev=/dev/sda`
-* `coreos.inst.stream=stable`
-* `coreos.inst.ignition_url=http://example.com/config.ign`
-. Press `<ENTER>` (isolinux) or `<CTRL-x>` (grub) to start the installation.
-
-Once the installation is complete, the machine reboots. After rebooting, the first boot process begins. It is at this time that Ignition ingests the configuration file and provisions the system as specified.
+. Once booted, you can fetch your Ignition config and run `coreos-installer`:
```

Comment:
I think maybe we should indicate a bit more about what is going on.

*The machine will boot and run Fedora CoreOS live from memory and give you a bash prompt. No hard disks are mounted so you can interactively fetch your ignition config and run coreos-installer:

Reply:
This extra clarification is probably useful for users who have never booted a live ISO before and might not understand what it is.

Reply:
I took your suggestion in spirit, but tweaked it. ;)  I think it's probably worth a "live system" section in the future we can link to on more information about that.